src/driver.cpp

```diff
@@ -179,6 +188,10 @@ Napi::Promise PressKey(const Napi::CallbackInfo& info) {
 
   Napi::Array modifierArray = info[1].As<Napi::Array>();
   int count = info[2].As<Napi::Number>();
```

Comment:
Is the reason that we don't get the Int32Value of this number because the for loop is in `driver` rather than the OS-specific drivers? If so, is there a reason for the difference in the APIs for "press this mouse button x times" and "press this key x times"?

Reply:
ah yeah good catch. I think there's probably some overloading happening in `Napi` to make this work, but I'll add the `Int32Value` here as well to be more explicit + consistent.